packages/core/src/SandboxPool.ts

```diff
@@ -10,7 +10,7 @@ import { filter, flatMap, map, merge, tap, zip } from 'rxjs/operators';
 import { Disposable } from 'typed-inject';
 
 import { coreTokens } from './di';
-import LoggingClientContext from './logging/LoggingClientContext';
+import { LoggingClientContext } from './logging/LoggingClientContext';
```

Comment:
Couldn't it be imported from logging index?

Reply:
`SandboxPool` is one of the "dead code" classes I'll remove before marking this pull request as ready for review. Its replacement is the `TestRunnerPool`.

Thanks for asking though.

Reply:
This is now removed